larpandoracontent/LArTrackShowerId/TrackShowerIdFeatureTool.cc

```diff
@@ -8,26 +8,23 @@
 
```

Comment:
General comment. It seems like the attempt to address some indentation issues in this file haven't worked, there appear to be a lot of inconsistent indentations in this file.